src/ansible_navigator/__init__.py

```diff
@@ -1,2 +1,14 @@
-"""The ansible-navigator application."""
-from ._version import __version__  # noqa: F401
+"""The ansible-navigator application.
+
+This is a transitional configuration during the migration
+from the hardcoded __version__ in _version to
+version being populated during the build.
+"""
+
+try:
+    from ._version import __version__
+except ImportError:
+    try:
+        from ._version import version  # type: ignore[attr-defined]
```

Comment:
Could we avoid the type ignore by setting `version = __version__` in `_version` for now?

(yo dawg, I heard you like versions, so I put `version` in your `_version` so you can `version` while you `__version__`) ???

Reply:
oh, that's an interesting idea..........  I think that might work.

Reply:
It's a bad idea. If this import succeeds, then there will be no `__version__` exposed in `__init__.py` which will cascade into other parts trying to access it.